lib/githubactions/token_validator_test.go

```diff
@@ -43,8 +43,8 @@ func fakeGithubIDP(t *testing.T) (*httptest.Server, jose.Signer) {
 	)
 	require.NoError(t, err)
 
-	url := ""
 	providerMux := http.NewServeMux()
+	srv := httptest.NewServer(providerMux)
```

Comment:
Same as above, don't forget to shut down the server.

Reply:
a8c047cd6f